app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,9 @@
     <%= content_for :head %>
   </head>
   <body>
+    <% if notice.present? %>
+      <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+    <% end %>
```

Comment:
I moved this flash notice up to the layout because messages were getting swallowed. It looks terrible, but I'm vaguely aware that @krschacht is working on styles? I'm happy to spruce it up if needed.

Anyway, with the notices getting swallowed, I wasn't able to write a helper method #login_user for the system integration tests and know that the login succeeded.

I'm happy to leave this as-is (minimally working) or to finish the refactor and make the notifications work system wide.

Reply:
This seems like a fine place for now